src/components/Book.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Book.css';
+
+const Book = (props) => {
+  const { book } = props;
+
+  return (
+    <div className="panel">
+      <div className="first">
+        <h5>{book.category}</h5>
+        <h2>{book.title}</h2>
+        <p>Suzanne Collins</p>
```

Comment:
**[OPTIONAL BUT VERY IMPORTANT TO CONSIDER AS A REACT DEVELOPER]**
I just noticed now. But I don't think it is correct to put static data in our components. We do have some `initialState` data in `src/index.js` right?  We could have set the individual book authors there (for each book). What is happening now is that whenever I add a new book the author will still be Suzanne Collins even if I am entering a Robert Kiyosaki book.